python/cudf/cudf/_lib/pylibcudf/__init__.pxd

```diff
@@ -48,6 +48,7 @@ __all__ = [
     "concatenate",
     "copying",
     "datetime",
+    "expressions",
```

Comment:
Was this supposed to add “experimental”?

Reply:
I chose not to originally because I didn't think we'd be using it anytime, but may as well add it for completeness. I've done so now.